port/port_objects.mk

```diff
@@ -55,6 +55,7 @@ ifeq (zos,$(OMR_HOST_OS))
 
   ifeq (1,$(OMR_ENV_DATA64))
     # 64-bit only
+    OBJECTS += omrcel4ro31
```

Comment:
You will need to make identical changes in https://github.com/eclipse/omr/blob/master/port/CMakeLists.txt. OMR zOS PR builds use CMake.

Reply:
Fixed under: https://github.com/eclipse/omr/pull/5834/commits/a1009000d43a8e34b7e64b680d1ee7b736dd4f30#diff-d332a50380ce5ae69404249c505db660bd6c238df98c0abeaa341f97199ab6d5R96  Thanks!